src/cerebral.d.ts

```diff
@@ -0,0 +1,107 @@
+﻿declare module "cerebral" {
```

Comment:
Publishing to @types is usually a fallback way for projects that doesn't provide typings right in npm package for any reason. Typings provided in npm package should be "import / export style", not "declare module style".
I think we should provide normal as-is typings right with cerebral2 and should be another package with your wrapping helpers and extended typings. That way it wouldn't require to install anything from @types